man/deepsea-monitor.1

```diff
@@ -0,0 +1,66 @@
+.TH deepsea-monitor 1 "DeepSea CLI man page"
+.SH NAME
+deepsea-monitor - Starts the stage execution monitor.
+
+.SH SYNOPSIS
+deepsea monitor [--help] [--show-dynamic-steps] [--show-state-steps]
+                [--simple-output]
+
+.SH DESCRIPTION
+Starts DeepSea progress monitor.
```

Comment:
"Starts the DeepSea stage execution progress monitor." ? (to unify the description throughout the manpage)